app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,36 @@
+class PostsController < ApplicationController
+  expose_decorated :post
+
+  before_action :authorize_resource
+
+  def show
```

Comment:
Should we add `index` action to be able to show all articles on the main page? Or it will be done within another story?